www/md_base/src/app/common/directives/inspectdata/inspectdata.tpl.jade

```diff
@@ -0,0 +1,13 @@
+div.inspect-data
+  div.row(layout="row", ng-repeat="item in inspectdata.items")
+    div.key(flex="33") {{ item.key }}
+    div.value(flex="66")
+      span(ng-if="item.value.type == 'string'") {{ item.value.value }}
+      a(ng-if="item.value.type == 'link'", ng-href="{{ item.value.value }}") {{ item.value.value }}
+      div.object-value(
+        ng-if="item.value.type == 'object'",
+        ng-init="collapsed = 1",
+        ng-click="collapsed = !collapsed",
```

Comment:
the problem with putting ng-click here is that you cannot easily copy paste the value, and you actually dont know that this is even clickable. So I would prefer you stick with something like this for expansion:

![image](https://cloud.githubusercontent.com/assets/109859/7278462/7d81803c-e8ca-11e4-89a1-ab3791c73562.png)